### ISSUE TITLE
Button upload component

### ISSUE THREAD

**Issue by djalmaaraujo** [Automattic/vip-design-system#282]
Currently, we are using a customized solution that it does not work properly for A11Y. We need to investigate a good way to build an Upload button component for us.

Reference: https://carbondesignsystem.com/components/file-uploader/usage/
